### DISCUSSION TITLE
Welcome to API Center discussions!

### DISCUSSION THREAD

**Discussion by mikebudzynski** [Azure/api-center#1] [Announcements]
## 👋 Welcome!

We’re using Discussions as a place to connect with API Center preview users. You're welcome to ask questions, share ideas, and engage with others. If you're looking to report a bug or request a feature, use *Issues* instead.

**Comment by ratheeshps** [comment]
Thanks Mike

**Reply by mikebudzynski** [reply]
@ratheeshps You can find instructions for getting access to preview here: https://aka.ms/apicenter/joinpreview. This form was also linked in the preview announcement blog post: https://aka.ms/apicenter/preview/blog.

**Comment by bendixon-zelis** [comment]
Thanks for the note, Mike. Looking forward to exploring the API Center features!
